src/v/cluster/commands.h

```diff
@@ -154,6 +156,12 @@ using delete_data_policy_cmd = controller_command<
   delete_data_policy_cmd_type,
   model::record_batch_type::data_policy_management_cmd>;
 
+using create_materialized_topic_cmd = controller_command<
+  model::topic_namespace,
```

Comment:
why both key and value are the same here ?. If you want to have both `source and materialized topic` it would be better to enclose that in a structure to add some context and semantics now it is just `key` `value` pair and to now what that mean one have to read `apply` method.